### DISCUSSION TITLE
When can we expect to see an update to 8.3 ?

### DISCUSSION THREAD

**Discussion by oywino** [LMS-Community/slimserver#603] [General]
Since I updated to 8.2, I've had numerous notifications that various fixes of "this" & "that" has been applied over time.
When do we "container" users get to enjoy all this hard work laid down by the maintainer?

Rgds

**Comment by michaelherger** [comment]
Just update your container and you'll get all those latest and greatest changes. As 8.2 hasn't been released as the stable release yet, there are new builds available as soon as there are changes.

**Comment by terual** [comment]
What tag are you using (assuming you are using docker)? If you are using 8.2.0 or dev you immediately should get the latest changes.
